### DISCUSSION TITLE
Code generation not stopping at repeating code that already exists

### DISCUSSION THREAD

**Discussion by kaetemi** [community/community#49136] [Copilot Conversations] [answered]
### Select Topic Area

Bug

### Body

![image](https://user-images.githubusercontent.com/1581053/222873414-be316edf-d760-4a70-aad0-6b2d47708bc3.png)

I'm getting suggestions that should be only one or a couple of lines generate entire blocks that are reverse repetitions of code that's already following.

Aside from getting that fixed, is there any hotkey to only accept a single line at a time instead of the whole suggested block? That'd be much more efficient with large suggestions.

**Comment by iamcjwilliams** [comment, marked as answer]
@kaetemi Have you taken a look here? https://docs.github.com/en/copilot/troubleshooting-github-copilot

**Reply by kaetemi** [reply]
Ah, thanks. Plugin was outdated and throwing errors in the logs.
